docs/2.9.0/docs/canton/usermanual/upgrading.rst

```diff
@@ -605,46 +683,78 @@ the upgrade test to understand the necessary downtime required for the migration
 Furthermore, upon reconnecting, the participant needs to recompute the new set of commitments. This can take
 a while for large numbers of contracts.
 
-Soft Synchronization Domain Connection Upgrade
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. _one_step_migration:
 
-.. note ::
+One-Step Migration
+------------------
+A *one-step migration* intends to :ref:`upgrade the binary <upgrade_canton_binary>` and
+:ref:`change the protocol version <change_pv>` in one step. It is a combination of the
+steps documented in :ref:`upgrade_canton_binary` and the :ref:`hard_sync_domain_upgrade`.
+Thus, strong understanding of these sections is required.
 
-    The soft sync domain connection upgrade is currently only supported as an alpha feature.
+The one-step migration covers an upgrade from Canton version 2.3 and following minor versions
+up to this minor release version. Additionally, it upgrades the protocol version supported by
+these prior releases to a protocol version supported by this minor release version.
 
-The hard sync domain connection upgrade requires coordination among all participants in a network. The
-soft sync domain connection upgrade is operationally much simpler, and can be leveraged using multi-sync-domain
-support (which exists as a pre-alpha feature only for now). By turning off non-unique contract keys,
-participants can connect to multiple sync domains and transfer contracts between sync domains. This allows us to avoid using the ``repair.migrate_domain`` step.
+.. note::
 
-Assuming the same setup as before, where the participant is connected to the old sync domain,
-we can just connect it to the new sync domain
+    There is no need for a one-step migration when upgrading from one release version to another,
+    as long as both release versions support the protocol version that your current sync domain uses.
+    Follow the steps for :ref:`upgrading the Canton binary <upgrade_canton_binary>` instead.
```

Comment:
Add reference to the table which is in https://docs.daml.com/canton/usermanual/versioning.html#canton-protocol-version ?